master/buildbot/data/codebase_commits.py

```diff
@@ -99,10 +99,7 @@ class CodebaseCommit(base.ResourceType):
     name = "commit"
     plural = "commits"
     endpoints = [CodebaseCommitEndpoint, CodebaseCommitByRevisionEndpoint, CodebaseCommitsEndpoint]
-    eventPathPatterns = """
-        /commits/:commitid
-        /codebases/:codebaseid/commits/:commitid
-    """
+    eventPathPatterns = ["/commits/:commitid", "/codebases/:codebaseid/commits/:commitid"]
```

Comment:
Add comma after the last element, so that lists are always formatted in one element per line.